R/pkg/R/SQLContext.R

```diff
@@ -328,6 +328,7 @@ setMethod("toDF", signature(x = "RDD"),
 #' It goes through the entire dataset once to determine the schema.
 #'
 #' @param path Path of file to read. A vector of multiple paths is allowed.
+#' @param ... additional external data source specific named properties.
```

Comment:
right - when 2 functions share the same @rdname, they are documented on the same Rd page and CRAN check requirements are to have 1 and only 1 `@param ...` when either function has `...` as parameter.

I haven't check, but my guess is you need to add `@param ...` for `@rdname read.json` since it is new.